civiclens/nlp/pipeline.py

```diff
@@ -96,7 +95,7 @@ def docs_have_titles():
 
     title_creator = titles.TitleChain()
     labeler = LabelChain()
-    sbert_model = SentenceTransformer("all-mpnet-base-v2")
+    sbert_model = sentence_transformer
```

Comment:
probably should just pass the `sentence_transformer` variable where needed instead of redefining a new variable

Reply:
agreed